README.md

```diff
@@ -292,10 +289,273 @@ client = Elastic::AppSearch::Client.new(host_identifier: 'host-c5s2mj', api_key:
 client.search('national-parks-demo', 'everglade')
 ```
 
+#### Log click-through
+
+Logging a click through
+
+```ruby
+engine_name = 'favorite-videos'
+options = {
+  :query => 'cat videos',
+  :document_id => 'INscMGmhmX4',
+  :request_id => 'e4c4dea0bd7ad3d2f676575ef16dc7d2',
+  :tags => ['firefox', 'web browser']
+}
+
+client.log_click_through(engine_name, options)
+```
+
+#### Analytics - Number of clicks-throughs received by a document
+
+```ruby
+engine_name = 'favorite-videos'
+options = {
+  :query => 'cats',
+  :page => {
+    :size => 20,
+  },
+  :filters => {
+    :date => {
+      :from => '2019-04-11T00:00:00+00:00',
+      :to => '2019-04-13T00:00:00+00:00'
+    }
+  }
+}
+
+client.get_top_clicks_analytics(engine_name, options)
+```
+
+#### Analytics - Queries, number of queries, and clicks received
+
+```ruby
+engine_name = 'favorite-videos'
+options = {
+  :page => {
+    :size => 20
+  },
+  :filters => {
+    :date => {
+      :from => '2019-04-11T00:00:00+00:00',
+      :to => '2019-04-13T00:00:00+00:00'
+    }
+  }
+}
+
+client.get_top_queries_analytics(engine_name, options)
+```
+
+#### Analytics - Number of clicks and total number of queries
+
+```ruby
+engine_name = 'favorite-videos'
+options = {
+  :filters => {
+    :all => [
+      {
+        :tag => ['mobile', 'web']
+      },{
+        :query => 'cats'
+      }, {
+        :document_id => '163'
+      }, {
+        :date => {
+          :from => '2018-07-05T12:00:00+00:00',
+          :to => '2018-07-05T14:00:00+00:00'
+        }
+      }
+    ]
+  },
+  :interval => 'hour'
+}
+
+client.get_count_analytics(engine_name, options)
+```
+
+#### Creating Synonym Sets
+
+```ruby
+engine_name = 'us-national-parks'
+
+client.create_synonym_set(engine_name, :synonyms => ['park', 'trail'])
+```
+
+#### Retrieving Synonym Sets
+
+```ruby
+engine_name = 'us-national-parks'
+
+client.get_synonym_set(engine_name, 'syn-5d8e6b5d40caae7dcb6e1b9c')
+```
+
+#### Listing Synonym Sets
+
+```ruby
+engine_name = 'us-national-parks'
+
+client.list_synonym_sets(engine_name, :current => 1, :size => 20)
+```
+
+#### Updating Synonym Sets
+
+```ruby
+engine_name = 'us-national-parks'
+
+client.update_synonym_set(engine_name, 'syn-5d8e6b5d40caae7dcb6e1b9c', :synonyms => ['park', 'trail', 'ground'])
+```
+
+#### Destroying Synonym Sets
+
+```ruby
+engine_name = 'us-national-parks'
+
+client.destroy_synonym_set(engine_name, 'syn-5d8e6b5d40caae7dcb6e1b9c')
+```
+
+#### Listing Credentials
+
+```ruby
+client.list_credentials(:current => 1, :size => 20)
+```
+
```

Comment:
We need to document the new get_credential method

Reply:
https://github.com/elastic/app-search-ruby/pull/16/commits/fd8f37435e75bd8452416aa267bfc9b734975fe7